packages/react-composites/src/composites/MeetingComposite/MeetingCallControlBar.tsx

```diff
@@ -95,7 +95,7 @@ export const MeetingCallControlBar = (props: MeetingCallControlBarProps): JSX.El
         </CallAdapterProvider>
       </Stack.Item>
       {meetingCallControlOptions !== false && (
-        <Stack.Item>
+        <Stack horizontal>
```

Comment:
can this be removed from this PR? What does this change do?

Reply:
this ensures that the chat and people buttons are aligned horizontally with each other instead of getting stacked on top of one another